HealthKitReporter/Classes/Model/HKUnitConvertable.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  HKUnitConvertable.swift
+//  
+//
+//  Created by Vignesh J on 16/02/21.
+//
+
+import Foundation
+import HealthKit
+
+public protocol HKUnitConvertable {
```

Comment:
I would keep the naming convention and remove the HK prefix, so the protocol name would be just "UnitConvertable"